src/config.c

```diff
@@ -312,7 +312,7 @@ CONFIG_GETTER(getForkGcRetryInterval) {
 
 // FORK_GC_CLEAN_NUMERIC_EMPTY_NODES
 CONFIG_SETTER(setForkGCCleanNumericEmptyNodes) {
-  config->forkGCCleanNumericEmptyNodes = 1;
+  config->forkGCCleanNumericEmptyNodes = !config->forkGCCleanNumericEmptyNodes;
```

Comment:
Maybe I missed it before but all our configs should be a key value so lets change it (and make sure to keep backward compatibility to what we already did).

Reply:
@MeirShpilraien add `_FORK_GC_CLEAN_NUMERIC_EMPTY_NODES` as we should not document it and just use it in case we later find a bug.